tests/tasks/setup/kit-cluster/default.yaml

```diff
@@ -81,25 +84,62 @@ spec:
               - name: scheduler
                 args:
                   - --kube-api-qps=$(params.scheduler-qps)
+        etcd:
+          spec:
+            nodeSelector: 
+              node.kubernetes.io/instance-type: $(params.cp-instance-type)
+            containers: 
+            - name: etcd
+              args:
+                - --quota-backend-bytes=8589934592
       EOL
+      aws sts get-caller-identity
+      #Use service account attached to pod
+      # TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+      # kubectl config set-credentials tekton --token=$TOKEN
+      # kubectl config set-context --current --user=tekton
+      #validate 
+      kubectl get pods -A
       kubectl create -f $(workspaces.config.path)/cp.yaml
-
-      cat > "$(workspaces.config.path)/dp.yaml" <<EOL
+      nodes=$(params.node_count)
+      #TODO: Assumption is always nodecount is greater than or equal to 1000 for scalability testing purposes.
+      asgs=$((nodes/1000))
+      asg_name=$(params.cluster-name)-nodes
+      for i in $(seq 1 $asgs)
+      do
+      filename="$(workspaces.config.path)/dp$i"."yaml"
+      echo $filename
+      cat > "$(workspaces.config.path)/dp$i"."yaml"<<EOL
       apiVersion: kit.k8s.sh/v1alpha1
       kind: DataPlane
       metadata:
-        name: $(params.cluster-name)-nodes 
+        name: "$asg_name-$i"
       spec: 
         clusterName: $(params.cluster-name) # Desired Cluster Name
-        nodeCount: $(params.node-count)
+        nodeCount: 1000 #Max nodes we can accomodate per ASG in KIT today
+        subnetSelector:
+          kit/hostcluster: "$(params.host-cluster-name)-dataplane"
+        allocationStrategy: "flexible"
+        instanceTypes: #defaulting to nitro-instances for now
+          - m5.2xlarge
+          - m5.4xlarge
+          - m4.2xlarge
+          - m4.4xlarge
```

Comment:
Why not m5.large or xlarge, 2xlarge and 4xlarge for scale testing seems to big when creating 10K+ nodes.
Also why are we defaulting to nitro instances, is there any benefit?

Reply:
can the data plane instance types be parameterized?

Reply:
I was thinking of parameterizing it in future PR's. Let's take that out of scope for this PR. 
I defaulted to min `2x` large because we can go up to `pod-denisty` of 50 or so if we want to in our testing :D 
I will tell you offline why I'm going for  specifically `nitro-instances` 😉